src/modules/ekf2/EKF/common.h

```diff
@@ -66,10 +66,11 @@ using math::Utilities::sq;
 using math::Utilities::updateYawInRotMat;
 
 // maximum sensor intervals in usec
-#define BARO_MAX_INTERVAL (uint64_t)2e5 ///< Maximum allowable time interval between pressure altitude measurements (uSec)
-#define EV_MAX_INTERVAL   (uint64_t)2e5 ///< Maximum allowable time interval between external vision system measurements (uSec)
-#define GPS_MAX_INTERVAL  (uint64_t)5e5 ///< Maximum allowable time interval between GPS measurements (uSec)
-#define RNG_MAX_INTERVAL  (uint64_t)2e5 ///< Maximum allowable time interval between range finder  measurements (uSec)
+#define BARO_MAX_INTERVAL     (uint64_t)2e5  ///< Maximum allowable time interval between pressure altitude measurements (uSec)
+#define EV_MAX_INTERVAL       (uint64_t)2e5  ///< Maximum allowable time interval between external vision system measurements (uSec)
+#define GNSS_MAX_INTERVAL     (uint64_t)5e5  ///< Maximum allowable time interval between GNSS measurements (uSec)
+#define GNSS_YAW_MAX_INTERVAL (uint64_t)15e5 ///< Maximum allowable time interval between GNSS yaw measurements (uSec)
+#define RNG_MAX_INTERVAL      (uint64_t)2e5  ///< Maximum allowable time interval between range finder  measurements (uSec)
```

Comment:
Might as well use types as well.

```suggestion
static constexpr uint64_t RNG_MAX_INTERVAL = 200'000; ///< Maximum allowable time interval between range finder  measurements (uSec)
```